cmd/quadlet/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsUnambiguousName(t *testing.T) {
+	tests := []struct {
+		input string
+		res   bool
+	}{
+		// Ambiguous names
+		{"fedora", false},
+		{"fedora:latest", false},
+		{"library/fedora", false},
+		{"library/fedora:latest", false},
+		{"busybox@sha256:d366a4665ab44f0648d7a00ae3fae139d55e32f9712c67accd604bb55df9d05a", false},
```

Comment:
Maybe
```suggestion
				// Note that digested references are not truly ambiguous (= insecure); we warn against short-name digested references to avoid the performance overhead of looking up short names.
				{"busybox@sha256:d366a4665ab44f0648d7a00ae3fae139d55e32f9712c67accd604bb55df9d05a", false},
```
or maybe find some other name for the function (or split it?), see elsewhere.